pytorch_transformers/modeling_roberta.py

```diff
@@ -46,22 +46,46 @@
 }
 
 
-class RobertaEmbeddings(BertEmbeddings):
+class RobertaEmbeddings(nn.Module):
     """
     Same as BertEmbeddings with a tiny tweak for positional embeddings indexing.
     """
     def __init__(self, config):
-        super(RobertaEmbeddings, self).__init__(config)
-        self.padding_idx = 1
+        super(RobertaEmbeddings, self).__init__()
+        self.padding_idx = config.padding_idx
+        self.word_embeddings = nn.Embedding(config.vocab_size, config.hidden_size, 
+                                            padding_idx=config.padding_idx)
+        self.position_embeddings = nn.Embedding(self.padding_idx + config.max_position_embeddings + 1,
```

Comment:
I think it is unnecessary to add `padding_idx` and `1` to `max_position_embeddings` because `max_position_embeddings` is already set to be 514 in the uploaded RoBERTa configuration.

```
In [1]: from pytorch_transformers import RobertaConfig
In [2]: RobertaConfig.from_pretrained('roberta-base').max_position_embeddings
Out[2]: 514```

Reply:
This is a definition issue. However, It is not general. 
In my case, I'm trying to train the model with my own vocab(I usually use 0 as `padding_idx`) and data.  And also I want to make it compatible with `max_seq_length`. In the RoBERTa's case, `514` doesn't mean it could receive actual `514` length input because of `padding_idx`.